src/test/java/com/qamp/placelab/test/LoginWithValidCredentials.java

```diff
@@ -1,66 +1,58 @@
 package com.qamp.placelab.test;
 
+import com.qamp.placelab.pages.LoginPage;
 import com.qamp.placelab.utills.WebDriverSetup;
 import org.openqa.selenium.By;
 import org.openqa.selenium.WebDriver;
 import org.testng.Assert;
 import org.testng.annotations.*;
+import org.testng.asserts.SoftAssert;
 
 public class LoginWithValidCredentials {
     private WebDriver driver;
+    private SoftAssert softAssert = new SoftAssert();
+    private LoginPage loginPage;
 
     @Parameters("browser")
     @BeforeTest
-    public void setup(@Optional("chrome")final String browser) {
+    public void setup(@Optional("chrome") final String browser) {
         driver = WebDriverSetup.getWebDriver(browser);
         driver.get("https://demo.placelab.com/");
         System.out.println("You opened  browser " + browser);
+        this.loginPage = new LoginPage(driver);
     }
 
-    @Parameters( { "email", "password" } )
-    @Test
+    @Parameters({"email", "password"})
+    @Test(description = "User is  able to log in with valid credentials ")
     public void loginWithValidEmailPassword(final String email, final String password) {
 
+        //validate log in page
+        loginPage.validateLogInPageContent();
 
-        final String expectedAdminUserRole = "Group Admin";
-        Assert.assertTrue(
-                driver.findElement(By.id("login_form")).isDisplayed(),
-                "Validate login from is displayed"
-        );
-
-        //enter a email
-        driver.findElement(By.id("email")).sendKeys(email);
-
-
-        //enter a password
-        driver.findElement(By.id("password")).sendKeys(password);
-
-        //submit
-        driver.findElement(By.name("commit")).click();
+        //enter credentials
+        loginPage.enterCredentials(email, password);
         try {
             Thread.sleep(2000);
         } catch (InterruptedException e) {
             throw new RuntimeException(e);
         }
 
         // verify user role
+        final String expectedAdminUserRole = "Group Admin";
         final String actualUserRole = driver.findElement(By.id("user-role")).getText();
```

Comment:
this should be a part of the page class, we should avoid using driver class here, all the implementations should be hidden in the pages